src/web/views/locales/en/terms-and-conditions.njk

```diff
@@ -0,0 +1,36 @@
+<div class="govuk-warning-text">
+    <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
```

Comment:
I think there is a nunjucks macro for this https://design-system.service.gov.uk/components/warning-text/

Reply:
done